src/ucp/wireup/ep_match.h

```diff
@@ -28,9 +28,9 @@ typedef struct {
  * The expected/unexpected lists are *not* circular
  */
 typedef struct ucp_ep_match_entry {
-    ucs_list_link_t          exp_ep_q;        /* Endpoints created by API and not
+    ucs_hlist_head_t         exp_ep_q;        /* Endpoints created by API and not
```

Comment:
btw, why do they have `_q` postfix if it is list?

Reply:
i guess because in some way it's used like a queue, similar to tag matching queue (matching is done according to conn_sn)